lib/database.cpp

```diff
@@ -114,19 +114,19 @@ void Database::setAccountPickle(const QString &matrixId, const QByteArray &pickl
 
 void Database::clear(const QString &matrixId)
 {
-    QSqlQuery query;
+    QSqlQuery query(QSqlDatabase::database(QStringLiteral("Quotient")));
```

Comment:
Some `makeQuery()` one-liner function seems to be in order. Maybe `prepareQuery()` even.